pkg/addons/addons.go

```diff
@@ -279,6 +281,19 @@ func EnableOrDisableAddon(cc *config.ClusterConfig, name string, val string) err
 		return errors.Wrap(err, "command runner")
 	}
 
+	crName := cc.KubernetesConfig.ContainerRuntime
+	cr, err := cruntime.New(cruntime.Config{Type: crName, Runner: runner})
+	if err != nil {
+		return errors.Wrap(err, "container runtime")
+	}
+	runtimePaused, err := cluster.CheckIfPaused(cr, []string{"kube-system"})
+	if err != nil {
+		return errors.Wrap(err, "check paused")
+	}
+	if runtimePaused {
+		out.Styled(style.Shrug, `Can't enable addon on a paused cluster, please unpause the cluster firstly.`)
+		return errors.New("Can't enable addon on a paused cluster, please unpause the cluster firstly.")
```

Comment:
Both strings say `Can't enable addon` but the user might be trying to `disable` the addon and the message will confuse them. The text should match the action they're trying to take.

Reply:
Updated the code to correctly print enable/disable based on user's action.